.npmrc

```diff
@@ -0,0 +1,8 @@
+# removes notice from projects looking for funding
+fund=true
+# stops auto-checking for dependencies' issues
+audit=true
+# ignore peerDependencies contract - in preparation of migration to npm 8
+legacy-peer-deps=true
```

Comment:
Am I correct that these don't actually do anything because they are the default values, however the `loglevel=warn` actually accomplishes the desired suppression? Maybe we should remove these then?

Reply:
`loglevel=warn` is already suppressing the output of `fund` and `audit`, so if we choose to keep `loglevel=warn` we can remove the other two. I'll rebase the branch and remove the first commit of the PR so they go away.